packages/sources/polkadot-balance/test/integration/adapter.test.ts

```diff
@@ -130,4 +130,9 @@ describe('Balance Endpoint', () => {
     const response = await testAdapter.request({})
     expect(response.statusCode).toEqual(400)
   })
+
+  it('should return error (empty addresses)', async () => {
+    const response = await testAdapter.request({ addresses: [] })
+    expect(response.statusCode).toEqual(400)
+  })
```

Comment:
I noticed it in the other two but could you add the snapshot check here too
```suggestion
  it('should return error (empty addresses)', async () => {
    const response = await testAdapter.request({ addresses: [] })
    expect(response.statusCode).toEqual(400)
    expect(response.json()).toMatchSnapshot()
  })
```